openapi.yaml

```
@@ -12191,16 +12191,86 @@ components:
         region:
           type: string
     ClusterCredentials:
+      oneOf:
+        - $ref: '#/components/schemas/AwsClusterCredentials'
+        - $ref: '#/components/schemas/ScalewayClusterCredentials'
+        - $ref: '#/components/schemas/GenericClusterCredentials'
+      discriminator:
+        propertyName: job_type
```

should be `object_type` here 
```suggestion
        propertyName: object_type
```